CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+Unreleased
+----------
+### Added
+* Add enzyme intl helpers: mockIntl, mountWithIntl, shallowWithIntl, and renderWithIntl
```

Comment:
I think we could just say `Initial stable release`

Reply:
https://github.com/cerner/terra-enzyme-intl/pull/1/commits/8b6a55a08aef552bc2736fa0fe874d12b55c8648